src/main/kotlin/org/rust/lang/core/CompilerFeatures.kt

```diff
@@ -78,6 +78,7 @@ val INCLUSIVE_RANGE_SYNTAX = CompilerFeature("inclusive_range_syntax", ACCEPTED,
 val INFER_OUTLIVES_REQUIREMENTS = CompilerFeature("infer_outlives_requirements", ACCEPTED, "1.30.0")
 val IRREFUTABLE_LET_PATTERNS = CompilerFeature("irrefutable_let_patterns", ACCEPTED, "1.33.0")
 val ITEM_LIKE_IMPORTS = CompilerFeature("item_like_imports", ACCEPTED, "1.15.0")
+val LABEL_BREAK_VALUE = CompilerFeature("label_break_value", ACCEPTED, "CURRENT_RUSTC_VERSION")
```

Comment:
`CURRENT_RUSTC_VERSION` is the new mechanism for unstable features, which uses a placeholder. Not sure if this will break anything in the plugin (i.e. semver parsing?) :thinking:

Reply:
It breaks since the plugin expects that `SemVer.parseFromText` returns non-null value for each compiler feature version

I've made [refactoring](https://github.com/intellij-rust/intellij-rust/pull/9480) to support the new placeholder mechanism